js/preload/preloadMain.js

```diff
@@ -86,6 +86,20 @@ const throttledSetIsInMeetingStatus = throttle(1000, function (isInMeeting) {
 local.ipcRenderer.on('on-page-load', () => {
     snackBar = new SnackBar();
 
+    // Enable spellchecker only for main window until
+    // the underline memory leak is fixed on electron
+    // https://github.com/electron/electron/issues/15459
+    if (window.name === 'main') {
```

Comment:
is there a better way to check if it's the main window? Maybe using the parent property or something like that?

Reply:
No, I don't think there is any other way and this is being used elsewhere